core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java

```diff
@@ -165,10 +167,19 @@ void reconnect(String transactionServiceGroup) {
             return;
         }
         if (CollectionUtils.isEmpty(availList)) {
-            String serviceGroup = RegistryFactory.getInstance()
-                                                 .getServiceGroup(transactionServiceGroup);
-            LOGGER.error("no available service '{}' found, please make sure registry config correct", serviceGroup);
-            return;
+            String clusterName = RegistryFactory.getInstance().getServiceGroup(transactionServiceGroup);
+
+            if (StringUtils.isNotBlank(clusterName)) {
+                LOGGER.error("no available service found in cluster '{}{}{}', please make sure registry config correct",
+                        ConfigurationKeys.SERVICE_PREFIX,
```

Comment:
if use third registry center ,will not read this configuration item.

Reply:
I modified the log message,please review it